packages/consumer/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * as utils from "./utils";
+export * as methods from "./methods";
+export * as parser from "./parsers";
```

Comment:
I would rather reexport by name so we are certain we aren't exporting too much stuff (potential breaking changes)